clang/include/clang/Driver/Options.td

```diff
@@ -1349,6 +1349,19 @@ def fno_hip_emit_relocatable : Flag<["-"], "fno-hip-emit-relocatable">,
   HelpText<"Do not override toolchain to compile HIP source to relocatable">;
 }
 
+// Clang specific/exclusive options for OpenACC.
+def openacc_macro_override
```

Comment:
Why do you need this form? Is not EQ form is enough?

Reply:
It is for general consistency here, we typically provide both for many similar options.  We typically don't use the 'EQ' form for CC1 at all (since it requires driver string-appending), and users are often more comfortable with the EQ version.  I'm not attached to either, but it seems that it is sensible to provide both.